mappings/jn.mapping

```diff
@@ -0,0 +1 @@
+CLASS jn
```

Comment:
Why a mapping file that does nothing?

It seems this is the package-private OperatingSystem, so we need to see in which package this is used so we know where to put this.

Reply:
i removed this mappings file for now